models/comments.model.js

```diff
@@ -0,0 +1,21 @@
+const db = require("../db/connection.js")
+
+exports.deleteCommentByIdOnComments = (req) => {
+    const commentID = req.params.comment_id.toString();
+    
+    const sql = 
+    `
+        DELETE FROM comments
+            WHERE comment_id = $1;
+    `;
+    return db.query(sql, [commentID])
+        .then(result => {
+            if(result.rowCount === 0){
+                return {status: 204}
+            }else{
+                return {
+                    status: 204
+                }
```

Comment:
Here were returning status 204 either way, so the if else block isn't doing anything